internal/service/sagemaker/domain.go

```diff
@@ -680,7 +680,6 @@ func ResourceDomain() *schema.Resource {
 									"user_group": {
 										Type:         schema.TypeString,
 										Optional:     true,
-										Default:      sagemaker.RStudioServerProUserGroupRStudioUser,
```

Comment:
This is a potenially breaking change, @ewbankkit wdyt?

Reply:
@deepakbshetty lets remove this from this PR. i see there is a follow PR to remove this

Reply:
@DrFaust92 Removed.